include/actionlib/client/action_client.h

```diff
@@ -207,18 +207,26 @@ class ActionClient
 
   void initClient(ros::CallbackQueueInterface* queue)
   {
-    status_sub_   = queue_subscribe("status",   1, &ActionClientT::statusCb,   this, queue);
-    feedback_sub_ = queue_subscribe("feedback", 1, &ActionClientT::feedbackCb, this, queue);
-    result_sub_   = queue_subscribe("result",   1, &ActionClientT::resultCb,   this, queue);
+    // read parameters indicating publish/subscribe queue sizes
+    int pub_queue_size;
+    int sub_queue_size;
+    n_.param("actionlib_client_pub_queue_size", pub_queue_size, 1);
+    n_.param("actionlib_client_sub_queue_size", sub_queue_size, 10);
```

Comment:
@safrimus You are absolutely right. I apologize for waiting three days to respond to this, I didn't see it... Will fix right now.